tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -258,6 +258,12 @@ static cl::opt<SPIRV::BuiltinFormat> SPIRVBuiltinFormat(
         clEnumValN(SPIRV::BuiltinFormat::Global, "global",
                    "Use globals to represent SPIR-V builtin variables")));
 
+static cl::opt<bool> SPIRVUseOpenCLExtInstructionsForLLVMMathIntrinsic(
+    "spirv-use-ocl-for-llvm-math-intrinsic", cl::init(true),
```

Comment:
@svenvh may I ask you to take a look if this option is OK?

Reply:
I wonder if we can generalize the option?  If I understand correctly, you essentially want to control whether the translator is allowed to emit `OpenCL.std OpExtInst` instructions?

Reply:
Yeah, exactly. So is your suggesting to rename it or something different? We kinda can find counterparts of https://registry.khronos.org/SPIR-V/specs/unified1/OpenCL.ExtendedInstructionSet.100.html in https://llvm.org/docs/LangRef.html#standard-c-c-library-intrinsics , so would it be `spirv-use-std-ocl-for-llvm-std-intrinsics` ?

Reply:
Mmm, I see. I will spend some time thinking about it, I'd prefer not to add an option (and enable it in some compiler's frontend driver), only to remove it in a couple of weeks, replacing with a better solution :) So we can keep PR open for a while